lib/alchemy/resource_filter.rb

```diff
@@ -0,0 +1,39 @@
+module Alchemy
+  class ResourceFilter
+    attr_reader :name
+
+    def initialize(filter, resource_name)
+      @filter = filter
+      @name = filter[:name]
+      @resource_name = resource_name
+      @values = filter[:values].blank? ? [] : filter[:values]
```

Comment:
```suggestion
      @values = filter[:values].presence || []
```

should do the same trick